Source/Core/DolphinQt2/Config/Graphics/AdvancedWidget.cpp

```diff
@@ -142,62 +142,62 @@ void AdvancedWidget::OnEmulationStateChanged(bool running)
+      "unchecked.");
   static const char TR_LOAD_CUSTOM_TEXTURE_DESCRIPTION[] = QT_TR_NOOP(
-      "Load custom textures from User/Load/Textures/<game_id>/.\n\nIf unsure, leave this "
+      "Loads custom textures from User/Load/Textures/<game_id>/.\n\nIf unsure, leave "
+      "unchecked.");
+  static const char TR_CACHE_CUSTOM_TEXTURE_DESCRIPTION[] = QT_TR_NOOP(
+      "Caches custom textures to system RAM on startup.\n\nThis can require exponentially "
+      "more RAM but fixes possible stuttering.\n\nIf unsure, leave unchecked.");
+  static const char TR_DUMP_EFB_DESCRIPTION[] = QT_TR_NOOP(
+      "Dumps the contents of EFB copies to User/Dump/Textures/.\n\nIf unsure, leave "
       "unchecked.");
-  static const char TR_CACHE_CUSTOM_TEXTURE_DESCRIPTION[] =
-      QT_TR_NOOP("Cache custom textures to system RAM on startup.\nThis can require exponentially "
-                 "more RAM but fixes possible stuttering.\n\nIf unsure, leave this unchecked.");
-  static const char TR_DUMP_EFB_DESCRIPTION[] =
-      QT_TR_NOOP("Dump the contents of EFB copies to User/Dump/Textures/.\n\nIf unsure, leave this "
-                 "unchecked.");
-  static const char TR_DISABLE_VRAM_COPIES_DESCRIPTION[] =
-      QT_TR_NOOP("Disables the VRAM copy of the EFB, forcing a round-trip to RAM. Inhibits all "
-                 "upscaling.\n\nIf unsure, leave this unchecked.");
+  static const char TR_DISABLE_VRAM_COPIES_DESCRIPTION[] = QT_TR_NOOP(
+      "Disables the VRAM copy of the EFB, forcing a round-trip to RAM. Inhibits all "
+      "upscaling.\n\nIf unsure, leave unchecked.");
   static const char TR_INTERNAL_RESOLUTION_FRAME_DUMPING_DESCRIPTION[] = QT_TR_NOOP(
-      "Create frame dumps and screenshots at the internal resolution of the renderer, rather than "
-      "the size of the window it is displayed within. If the aspect ratio is widescreen, the "
-      "output "
+      "Creates frame dumps and screenshots at the internal resolution of the "
+      "renderer, rather than the size of the window it is displayed within. "
+      "\n\nIf the aspect ratio is widescreen, the output "
       "image will be scaled horizontally to preserve the vertical resolution.\n\nIf unsure, leave "
-      "this unchecked.");
+      "unchecked.");
 #if defined(HAVE_FFMPEG)
   static const char TR_USE_FFV1_DESCRIPTION[] =
-      QT_TR_NOOP("Encode frame dumps using the FFV1 codec.\n\nIf unsure, leave this unchecked.");
+      QT_TR_NOOP("Encodes frame dumps using the FFV1 codec.\n\nIf unsure, leave unchecked.");
 #endif
   static const char TR_FREE_LOOK_DESCRIPTION[] = QT_TR_NOOP(
-      "This feature allows you to change the game's camera.\nMove the mouse while holding the "
-      "right "
-      "mouse button to pan and while holding the middle button to move.\nHold SHIFT and press "
+      "Allows the user to manipulate the game camera.\n\nMove "
+      "the mouse while holding the right "
```

Comment:
I think this would be much more concise and readable:

Move the mouse and hold the right button to pan or middle button to move. Hold SHIFT and use the WASD keys to move the camera. Press SHIFT+1 to move faster or SHIFT+2 to move slower. Press SHIFT+R to reset the camera or SHIFT+F to reset the speed.

Reply:
That's much better, thanks!